### PR TITLE
HIVE-27483: HMS add_partitions audit log adds table name and metrics

### PR DESCRIPTION
**Pull request by cxzl25**
### What changes were proposed in this pull request?
Add table name to audit log and metrics.

### Why are the changes needed?
We are missing the table name and metrics related to `add_partitions` RPC.

### Does this PR introduce _any_ user-facing change?
No

### Is the change a dependency upgrade?
No

### CODE REVIEW

File: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java [hunk 1]
```diff
@@ -4445,7 +4444,20 @@ public int add_partitions_pspec(final List<PartitionSpec> partSpecs)
       catName = partSpecs.get(0).getCatName();
     }
 
-    return add_partitions_pspec_core(getMS(), catName, dbName, tableName, partSpecs, false);
+    Integer ret = null;
+    Exception ex = null;
+    try {
+      startTableFunction("add_partitions_pspec", catName, dbName, tableName);
```
Comment (aturoczy) [inline standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java:4450]: I would put the startTableFunction outside of the try block.

**Review by aturoczy — APPROVED** [review]
The code itself is OK.  I do not find test for the audit logs, maybe I missed it, but if it possible could you please check if there is any test around it? 
Also please move the jira ticket into the correct state. 

Thanks for the contribution.

**Comment by cxzl25** [comment]
> I do not find test for the audit logs, maybe I missed it, but if it possible could you please check if there is any test around it?

Yes, we are missing the audit log test.

> Also please move the jira ticket into the correct state.

Updated.

**Comment by aturoczy** [comment]
@nrg4878 Could you please check this PR?

**Comment by cxzl25** [comment]
Fixed by https://github.com/apache/hive/pull/4480
Thanks for your review @aturoczy

**Pull request closed without merging**